semgrep_output_v1.atd

```diff
@@ -1006,49 +1006,49 @@ type project_metadata = {
      * TODO: see Github_metadata.ml
```

Comment:
@zyannes would love some answers to the TODO comments I've put 10 lines above.

Reply:
This data is sent to the POST `/scans` endpoint when a scan is created and when getting the scan config.

Reply:
And a big +1 to splitting this into subtypes where `project_metadata` is a superset.

Reply:
FYI I don't think we are using the `project_metadata` type directly in the `/scans` endpoint today.